modules/caddyhttp/errors.go

```diff
@@ -115,6 +113,8 @@ func trace() string {
 	return ""
 }
 
+type errorCtxKeyT int32
```

Comment:
Super minor as there's only one of these, but IMO `struct{}` is better as a type for context keys - an `int32` is 4 bytes, whereas a `struct{}` is 0b. Also no need to use an arbitrary `0` with no real meaning... Something like:

```go
type errorCtxKeyT struct{}

...
const ErrorCtxKey = errorCtxKeyT{}
```

It's probably more a matter of taste than anything, though, so take it or leave it!

Reply:
Oh, right, I've thought about using structs but disregarded that since {} would have to be written everywhere. Somehow didn't think about doing pretty much the same thing as I did here though, lol. I'll change it to struct{} :)

Reply:
Ah, it's going to have to be `var` though, so probably not worth it.

Reply:
> Ah, it's going to have to be `var` though, so probably not worth it.

Ah, right. I suppose it matters in this case because `ErrorCtxKey` is exported. I'm used to avoiding exporting context keys to keep access limited. Too bad 😞